server/cmwell-sparql-agent/src/main/scala/cmwell/tools/data/sparql/SparqlProcessorManager.scala

```diff
@@ -64,27 +64,26 @@ case object CheckConfig
 case class AnalyzeReceivedJobs(jobsRead: Set[JobRead])
 
 sealed trait JobStatus {
+  val statusString : String
   val job: Job
   val canBeRestarted: Boolean = false
-  def statusString : String
 }
 
 sealed trait JobActive extends JobStatus {
-  override def statusString = " Active"
   val reporter: ActorRef
 }
 
-case class JobRunning(job: Job, killSwitch: KillSwitch, reporter: ActorRef) extends JobActive
-case class JobPausing(job: Job, killSwitch: KillSwitch, reporter: ActorRef) extends JobActive
-case class JobStopping(job: Job, killSwitch: KillSwitch, reporter: ActorRef) extends JobActive
+case class JobRunning(job: Job, killSwitch: KillSwitch, reporter: ActorRef, statusString : String = "Running") extends JobActive
+case class JobPausing(job: Job, killSwitch: KillSwitch, reporter: ActorRef, statusString : String = "Pausing") extends JobActive
+case class JobStopping(job: Job, killSwitch: KillSwitch, reporter: ActorRef, statusString : String = "Stopping") extends JobActive
```

Comment:
@nicklester Does `statusString` can take any other value than the given default?
If not, and it seems that way, it would be better to define the class as, e.g:
```scala
case class JobRunning(…) extends JobActive {
  override val statusString: String = "Running"
}
```
If in other places in code you want to pattern match and get the extracted string, please use extractors. e.g:
```scala
object JobActive {
  def unapply(job: JobActive): Option[(job: Job, killSwitch: KillSwitch, reporter: ActorRef, statusString: String)] = 
    Some((job.job,job.killSwitch,job.reporter,job.statusString))
}
…
job match {
  case JobActive(job, killSwitch, reporter, "Running") => …
  case JobActive(job, killSwitch, reporter, pausingOrStopping) => …
  …
}
```
Also, better yet, if strings are constants. let's use a sealed trait + case objects instead.

P.S.
If you use extractor, maybe you won't need `case class`es. 
You could use regular (`final`) classes instead.
Also, as a good practice, every `case class` should also be marked `final`